components/Evaluator/HackerList.js

```diff
@@ -0,0 +1,74 @@
+import React, { useState, useEffect } from 'react';
+import styled from 'styled-components';
+import HackerEntry from './HackerEntry';
+import Input from '../input';
+import { COLOR } from '../../constants';
+
+const Title = styled.span`
+  font-size: 15px;
+  line-height: 19px;
+  color: ${COLOR.MIDNIGHT_PURPLE};
+  font-weight: bold;
+`;
+
+export default function HackerList({
+  applicants,
+  selectedApplicant,
+  setSelectedApplicant = () => {},
+}) {
+  const [search, setSearch] = useState('');
+  const [debouncedSearch, setDebouncedSearch] = useState('');
+  const [filtered, setFiltered] = useState([]);
+
+  // debounce search
+  useEffect(() => {
+    const handler = setTimeout(() => {
+      setDebouncedSearch(search);
+    }, 20);
+
+    return () => {
+      // teardown
+      clearTimeout(handler);
+    };
+  }, [search]);
```

Comment:
nit: we wrote a utility debounced function - if that works here maybe we can use it to avoid duplicate code

Reply:
Where?

Reply:
oh shit my bad, it's in portal: https://github.com/nwplus/portal/blob/d176796f19634da4a9da8d1e0e271c6dc0b020f3/src/utility/utilities.js#L116-L134

If you wanna copy it over feel free to, but otherwise nbd